BeerMeUiKit/ViewController.swift

```diff
@@ -8,12 +8,79 @@
 import UIKit
 
 class ViewController: UIViewController {
-
-    override func viewDidLoad() {
-        super.viewDidLoad()
-        // Do any additional setup after loading the view.
+    
+    @IBOutlet weak var beerNameView: UILabel!
+    
+    @IBOutlet weak var beerImageView: UIImageView!
+    
+    @IBOutlet weak var beerBrandNameView: UILabel!
+    
+    @IBOutlet weak var alcholPercentageView: UILabel!
+    
+    @IBOutlet weak var beerStyleView: UILabel!
+    
+    @IBOutlet weak var beerMaltsView: UILabel!
+    
+    
+    var beer: BeerModel = BeerModel(id: 0, uid: "", brand: "", name: "", style: "", hop: "", yeast: "", malts: "", ibu: "", alcohol: "", blg: "")
+  
+    let notFoundImage = UIImage(resource: ._404ImageNotFound)
+        
+    @IBAction func newBeerButton(_ sender: Any) {
+        getBeers()
+    }
+    
+    func getBeers() {
+        print("inizio di getBeers")
```

Comment:
```suggestion
    func getBeers() {
```

meglio non committare i print utilizzati per debugging